src/App.js

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
No need to add extra lines here, code is short enough.
```suggestion
    <Sum a={2} b={3} />
```